src/rules/at-rule-blockless-newline-after/index.js

```diff
@@ -0,0 +1,60 @@
+import {
+  cssStatementHasBlock,
+  rawNodeString,
+  report,
+  ruleMessages,
+  validateOptions,
+  whitespaceChecker,
+} from "../../utils"
+
+export const ruleName = "at-rule-blockless-newline-after"
+
+export const messages = ruleMessages(ruleName, {
+  expectedAfter: () => "Expected newline after blockless at-rule",
+})
+
+export default function (actual) {
+  const checker = whitespaceChecker("newline", actual, messages)
+
+  return (root, result) => {
+    const validOptions = validateOptions(result, ruleName, {
+      actual,
+      possible: ["always"],
+    })
+    if (!validOptions) { return }
+
+    root.walkAtRules(statement => {
+
+      const isNested = statement.parent !== root
```

Comment:
I'm no sure we should be ignoring nested `@rules` by default. Shall we strip out this early return?